packages/static-wado-scp/lib/program/index.js

```diff
@@ -2,8 +2,16 @@ const staticWadoUtil = require("@ohif/static-wado-util");
 const { DcmjsDimseScp, Server } = require("..");
 const packageJson = require("../../package.json");
 
+const aeConfig = staticWadoUtil.aeConfig;
+
 function main() {
-  const port = this.dicomWebScpConfig.scpPort || 11112;
+  const { queryAe } = this.dicomWebScpConfig;
+  if (!queryAe) throw new Error("queryAe not specified");
```

Comment:
It is not a problem to not specify queryAe - that is a perfectly valid configuration, so you can't throw an error here, it just means it only returns local data, not proxying to another DIMSE system.